deploy/aws/README.md

```diff
@@ -0,0 +1,23 @@
+### AWS Deployment using SAM
+The AWS Serverless Application Model (SAM) extends AWS CloudFormation to provide a simplified way of defining the Amazon API Gateway APIs, AWS Lambda functions, and other AWS resources needed by your serverless application.
+
+##### Pre-requirements
+* lambda code should be packaged and uploaded to s3
+* template.yaml should be updated with lambda s3 location and proper handler name
+
+##### Deployment process
+To deploy for the first time, and for each update, run both of the following commands in order:
+```
+aws cloudformation package --template-file template.yaml \
+--output-template-file template-out.yaml
```

Comment:
I generally would like to see paths be relative to the root of the repo, so `--template-file template.yaml` should be `--template-file deploy/aws/template.yaml`.

Also when I run this using aws-cli/1.11.167 I get `aws: error: argument --s3-bucket is required`. (1.11.167 is latest per the [release notes](https://github.com/aws/aws-cli/blob/master/CHANGELOG.rst)).